dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```

Comment:
Please do not modify copyright of existing files

Reply:
The I don't get why there is a range

Reply:
there is a range for files that were created before the open sourcing of the code (between 2015 and 2019). For new files no need to have a range, current year is enough, but sometimes the range is kept (due to a copy paste for example) and it makes no harm. 
However we should not need to modify the copyright of existing files.

Reply:
OK